operator/Makefile

```diff
@@ -57,15 +57,15 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 
 .PHONY: vet
 vet: ## Run go vet against code.
-	go vet ./...
+	go vet -tags testing ./...
```

Comment:
hmm looking at this, a dedicated build tag also does not seem to be really beautiful. Do you know the best practice here? It would be good to stick to convention here and Ive never seen this style of test imports.

Reply:
I agree. But after reading this: https://stackoverflow.com/questions/31794141/can-i-create-shared-test-utilities
I think this is the better option

Reply:
Considering this and also some other posts, I would suggest we revert to your original suggestion but with slightly different naming as per recommendation of https://go.dev/doc/go1.4#internalpackages

In this, you should create an "internal/testing" package that cannot be used by other modules and will not be built by the runtime